backend/clickhouse/query.go

```diff
@@ -611,11 +616,13 @@ func readWorkspaceMetrics[T ~string](ctx context.Context, client *Client, sample
 		OrderForwardNatural,
 	)
 
-	var metricExpr string
-	if col, found := keysToColumns[T(strings.ToLower(column))]; found {
-		metricExpr = fmt.Sprintf("toFloat64(%s)", col)
-	} else {
-		metricExpr = fmt.Sprintf("toFloat64OrNull(%s[%s])", attributesColumn, fromSb.Var(column))
```

Comment:
i consolidated both to `toFloat64OrNull` - this means now that if a `keysToColumns` value is not numeric we would not error but drop those points from the aggregation, which seems safer to me